google-cloud-contrib/google-cloud-nio/src/test/java/com/google/cloud/storage/contrib/nio/CloudStorageFileSystemProviderTest.java

```diff
@@ -655,28 +654,26 @@ public void testNewFileSystem() throws IOException {
   }
 
   @Test
-  public void testFromSpace() throws UnsupportedEncodingException {
+  public void testFromSpace() throws UnsupportedEncodingException, IOException {
```

Comment:
For tests, I think it's easier to just do `throws Exception` (applies to multiple tests)